### PR TITLE
{bio}[foss/2021a] freebayes v1.3.6, vcflib v1.0.3, SeqLib v1.2.0 w/ Python 3.9.5

### PR DESCRIPTION
**Pull request by SIB-software**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 11 out of 11 (3 easyconfigs in total)
node3135.skitty.os - Linux RHEL 8.4, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/1538be22f51dffc06976e94087ea0963 for a full test report.

File: easybuild/easyconfigs/v/vcflib/vcflib-1.0.3-foss-2021a-R-4.1.0.eb [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+ described by VCF. The Variant Call Format (VCF) is a flat-file, tab-delimited textual format intended
+ to concisely describe reference-indexed genetic variations between individuals."""
+
+toolchain = {'name': 'foss', 'version': '2021a'}
+
+source_urls = [GITHUB_SOURCE]
+sources = ['v%(version)s.tar.gz']
+patches = [
+    '%(name)s-%(version)s_use-external-deps.patch',
+    'vcflib-1.0.3_fix-shared-linking.patch',
```
Comment (jfgrimm) [inline easybuild/easyconfigs/v/vcflib/vcflib-1.0.3-foss-2021a-R-4.1.0.eb:20]:
```suggestion
    '%(name)s-1.0.3_use-external-deps.patch',
    '%(name)s-1.0.3_fix-shared-linking.patch',
```

File: easybuild/easyconfigs/s/SeqLib/SeqLib-1.2.0-GCC-10.3.0.eb [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+description = """C++ interface to HTSlib, BWA-MEM and Fermi."""
+
+toolchain = {'name': 'GCC', 'version': '10.3.0'}
+
+github_account = 'walaj'
+source_urls = [GITHUB_SOURCE]
+sources = ['%(version)s.tar.gz']
+patches = [
+    '%(name)s-%(version_major_minor)s.0_avoid-bwa-fml-namespace-conflict.patch',
+    '%(name)s-%(version_major_minor)s.0_use-external-deps.patch',
```
Comment (jfgrimm) [inline easybuild/easyconfigs/s/SeqLib/SeqLib-1.2.0-GCC-10.3.0.eb:18]: templating the name, but hardcoding the patch version, generally makes it easier when creating easyconfigs for newer releases
```suggestion
    '%(name)s-1.2.0_avoid-bwa-fml-namespace-conflict.patch',
    '%(name)s-1.2.0_use-external-deps.patch',
```

**Review by jfgrimm — DISMISSED** [review]

File: easybuild/easyconfigs/v/vcflib/vcflib-1.0.3-foss-2021a-R-4.1.0.eb (hunk 1 above)
Comment (smoretti) [reply easybuild/easyconfigs/v/vcflib/vcflib-1.0.3-foss-2021a-R-4.1.0.eb:20]: Done

File: easybuild/easyconfigs/s/SeqLib/SeqLib-1.2.0-GCC-10.3.0.eb (hunk 2 above)
Comment (smoretti) [reply easybuild/easyconfigs/s/SeqLib/SeqLib-1.2.0-GCC-10.3.0.eb:18]: Done

**Comment by boegel** [comment]
@boegelbot please test @ jsc-zen2

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
Going in, thanks @SIB-software!

**Pull request merged**